server/game/core/ability/abilityTargets/CardTargetResolver.ts

```diff
@@ -27,6 +35,13 @@ export class CardTargetResolver extends TargetResolver<ICardTargetResolver<Abili
         this.validateLocationLegalForTarget(properties);
     }
 
+    private allZonesAreHidden(locationFilter): boolean {
+        if (Array.isArray(locationFilter)) {
```

Comment:
I always forget to use it but there is a helper function called `asArray()` in Helpers.ts that does this as a one-liner